src/index.ts

```diff
@@ -640,4 +641,15 @@ export function getContract(chainId: string, contractAddress: string): Promise<C
   })
 }
 
+export function getAuthNonce(): Promise<AuthNonce> {
+  return getEndpoint(baseUrl, '/v1/auth/nonce', { credentials: 'include' }, undefined)
```

Comment:
Do we need to pass `undefined` explicitly?
```suggestion
  return getEndpoint(baseUrl, '/v1/auth/nonce', { credentials: 'include' })
```